includes/Modules/Sign_In_With_Google.php

```diff
@@ -42,6 +43,7 @@ final class Sign_In_With_Google extends Module implements Module_With_Assets, Mo
 	 * @since 1.137.0
 	 */
 	public function register() {
+		add_action( 'login_message', array( $this, 'render_signin_button' ) );
```

Comment:
`login_message` is a filter, not an action.

Reply:
Correct, it does [support HTML rendering](https://developer.wordpress.org/reference/hooks/login_message/)

> This filter can return HTML markup.

Reply:
I moved back to `login_form` now that I'm positioning with JS.